### ISSUE TITLE
[cob_kinematics] remove?

### ISSUE THREAD

**Issue by fmessmer** [ipa320/cob_manipulation#72]
@ipa-mdl 
Is there anything about `cob_kinematics` package that is worth to be kept?

**Comment by mathias-luedtke** [comment]
Is there a reason to delete it?
It features a urdf2openrave converter.

**Comment by fmessmer** [comment]
I wasn't aware of that....maybe we should put that somewhere more prominent then!
Would this be helpful for generating COLLADA models for e.g. SDHX gripper in order to generate grasps with `cob_grasp_generation` (see #70)?

**Comment by fmessmer** [comment]
However, we do not compile it (https://github.com/ipa320/cob_manipulation/blob/indigo_dev/cob_kinematics/CMakeLists.txt#L22)
Does it still compile?

**Comment by fmessmer** [comment]
@ipa-mdl
Is the converter related or derived from any of these:
- http://wiki.ros.org/collada_urdf
- https://github.com/personalrobotics/or_urdf

**Comment by mathias-luedtke** [comment]
No, its unrelated.
However, https://github.com/personalrobotics/or_urdf seems to cover much more aspects..
My implementation just reads an URDF and outputs an OpenRAVE XML (http://openrave.programmingvision.com/wiki/index.php/Format:XML) with the kinematic structure.

**Comment by mathias-luedtke** [comment]
I just noticed that it does not have any install tags..

**Comment by fmessmer** [comment]
There is a `ìkfast_kinematics_plugin` in [moveit_kinematics](https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_kinematics/ikfast_kinematics_plugin) and a Tutorial [here](http://docs.ros.org/indigo/api/moveit_ikfast/html/doc/ikfast_tutorial.html)

IKFast requires OpenRAVE (it seems to be relying on version 0.8) which has to be installed manually anyway (no `rosdep` entry)

@ipa-mdl 
Do you think it is worth keeping this package then?
The yaml's in the config are out of date....although the generated cpp-files still might be usable....but I don't know....and we do not install anyway...and it hasn't been used since electric...:wink:

I vote for removal!

I guess the only additional features not included in moveit are
 - GetPositionIKExtended.srv
 - and the converter tool (which seems to be replacable by the tools mentioned in https://github.com/ipa320/cob_manipulation/issues/72#issuecomment-207733816)

**Comment by mathias-luedtke** [comment]
TL;DR: +1 for removal

> GetPositionIKExtended.srv

This is used in 3 old python scripts https://github.com/search?q=org%3Aipa320+GetPositionIKExtended&type=Code
But the providing node is gone anyway.

> and the converter tool

It is probably outdated as well.

**Comment by fmessmer** [comment]
removed for kinetic
